hera_gpu/vis.py

```diff
@@ -0,0 +1,260 @@
+import pycuda.autoinit
+from pycuda import compiler, gpuarray, driver
+from skcuda.cublas import cublasCreate, cublasSetStream, cublasSgemm, cublasCgemm, cublasDestroy, cublasDgemm, cublasZgemm
+import numpy as np
+from math import ceil
```

Comment:
Is there a reason to use this instead of `np.ceil`?

Reply:
Sure.